jenkins/opensearch/check-for-build.jenkinsfile

```diff
@@ -0,0 +1,86 @@
+lib = library(identifier: "jenkins@20211123", retriever: legacySCM(scm))
+
+pipeline {
+    agent none
+    options {
+        disableConcurrentBuilds() // Make sure not to interupt builds in progress
+    }
+    triggers {
+        parameterizedCron '''
+            H */12 * * * %INPUT_MANIFEST=1.3.0/opensearch-1.3.0.yml;TARGET_JOB_NAME=distribution-build-opensearch
+            H 1 * * * %INPUT_MANIFEST=2.0.0/opensearch-2.0.0.yml;TARGET_JOB_NAME=distribution-build-opensearch
+        '''
+    }
+    parameters {
+        string(
+            name: 'INPUT_MANIFEST',
+            description: 'Input manifest under the manifests folder, e.g. 2.0.0/opensearch-2.0.0.yml.',
+            trim: true
+        )
+        string(
+            name: 'TARGET_JOB_NAME',
+            description: 'Job to trigger if build has changed',
+            trim: true
+        )
+    }
+    stages {
+        stage('detect docker image + args') {
+            agent {
+                docker {
+                    label 'Jenkins-Agent-al2-x64-c54xlarge-Docker-Host'
+                    image 'opensearchstaging/ci-runner:centos7-x64-arm64-jdkmulti-node10.24.1-cypress6.9.1-20211028'
```

Comment:
Why is this one specifically pointing to a image but `check-sha-exists` stage doesnt?

Reply:
This job selects the docker image for the build process, this job needs to load a specific image, this is following the standard practices for our jenkinsfiles